apps/pollbook/backend/src/store.ts

```diff
@@ -348,27 +354,55 @@ export abstract class Store {
     return this.election;
   }
 
+  getMachineInformation(): PollbookInformation | undefined {
+    const row = this.client.one(
+      `
+          select election_data, ballot_hash, package_hash
+          from elections
+          order by rowid desc
+          limit 1
+        `
+    ) as { election_data: string; ballot_hash: string; package_hash: string };
+    if (!row) {
+      return undefined;
+    }
+    const election: Election = safeParseElection(
+      row.election_data
+    ).unsafeUnwrap();
+    return {
+      configuredElectionId: election.id,
+      configuredElectionName: election.title,
+      configuredElectionBallotHash: row.ballot_hash,
+      configuredPollbookPackageHash: row.package_hash,
+    };
+  }
+
   setElectionAndVoters(
-    election: Election,
+    electionDefinition: ElectionDefinition,
+    packageHash: string,
     validStreets: ValidStreetInfo[],
     voters: Voter[]
   ): void {
-    this.election = election;
+    this.election = electionDefinition.election;
     this.validStreetInfo = validStreets;
     this.client.transaction(() => {
       this.client.run(
         `
           insert into elections (
             election_id,
             election_data,
+            ballot_hash,
```

Comment:
Why is the ballot hash necessary if the pollbook isn't interacting with ballots?

Reply:
Leaving this for now but depending on what we decide in slack I can address this when I followup to finish the rest of the pollbook package hash work